docs/version0.24.md

```diff
@@ -71,6 +71,15 @@ The `-XX:[+|-]PortableSharedCache` option is now supported on IBM Z&reg; and POW
 The `-XX:[+|-]ShareAnonymousClasses` option currently enables and disables the storage of VM anonymous classes in the shared classes cache. From OpenJDK 15, this option now enables and disables the storage of hidden classes in the shared classes cache. See [`-XX:[+|-]ShareAnonymousClasses`](xxshareanonymousclasses.md) for more details about this option.
 
 
+### Additional parameters for `jcmd Dump` commands
+
+You can now include additional parameters for `jcmd Dump` commands to control both system and heap dumps. The new options are as follows:
```

Comment:
"to control both system and heap dumps" is incompatible with `Dump.system`, `Dump.heap`, `Dump.java`, `Dump.snap`, i.e. it doesn't mention java or snap dumps. Perhaps "to control both system and heap dumps" should just be removed.

`request=<parameter>` should refer to https://www.eclipse.org/openj9/docs/xdump/#requestltrequestsgt and `opts=<parameter>` to https://www.eclipse.org/openj9/docs/xdump/#optsltoptionsgt

Reply:
@pshipton ... updates made